website/docs/r/edgegateway_settings.html.markdown

```diff
@@ -0,0 +1,94 @@
+
+Supported in provider *v3.0+*
+
+## Example Usage
+
+```hcl
+data "vcd_edgegateway" "egw" {
+  name = "my-egw"
+}
+
+resource "vcd_edgegateway_settings" "egw-settings" {
+  edge_gateway_id         = data.vcd_edgegateway.egw.id
+  lb_enabled              = true
+  lb_acceleration_enabled = true
+  lb_logging_enabled      = true
+  lb_loglevel             = "debug"
+
+  fw_enabled                      = true
+  fw_default_rule_logging_enabled = true
+  fw_default_rule_action          = "accept"
+}
+```
+
+-> **Tip:** Although this resource changes values in the edge gateway referenced as a data source, due to how Terraform works, the state
+of the edge gateway doesn't get updated. To reconcile the state of the data source with the values modified in `vcd_edgegateway_settings`,
+you need to run `terraform refresh` after `apply`.
+
+=> **Note:** Although tenants can enable load balancing using this resource, they can't set the properties `lb_logging_enabled` and `lb_loglevel`.
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to which the VDC belongs. Optional if defined at provider level.
+* `vdc` - (Optional) The name of VDC that owns the edge gateway. Optional if defined at provider level. 
+* `edge_gateway_name` - (Optional) A unique name for the edge gateway. (Required if `edge_gateway_id` is not set)
+* `edge_gateway_id` - (Optional) The edge gateway ID. (Required if `edge_gateway_name` is not set)
+* `lb_enabled` - (Optional) Enable load balancing. Default is `false`.
+* `lb_acceleration_enabled` - (Optional) Enable to configure the load balancer.
+* `lb_logging_enabled` - (Optional) Enables the edge gateway load balancer to collect traffic logs.
+Default is `false`. Note: **only System administrators can change this property**. It is ignored by API for Org users.
+* `lb_loglevel` - (Optional) Choose the severity of events to be logged. One of `emergency`,
+`alert`, `critical`, `error`, `warning`, `notice`, `info`, `debug`. Note: **only System administrators can change this property**. It is ignored by API for Org users.
+* `fw_enabled` (Optional) Enable firewall. Default `true`.
+* `fw_default_rule_logging_enabled` (Optional) Enable default firewall rule (last in the processing 
+order) logging. Default `false`.
+* `fw_default_rule_action` (Optional) Default firewall rule (last in the processing order) action.
+One of `accept` or `deny`. Default `deny`.
+
+## Importing
+
+~> **Note:** The current implementation of Terraform import can only import resources into the state. It does not generate
+configuration. [More information.][docs-import]
+
+An existing edge gateway settings can be [imported][docs-import] into this resource via supplying its path. 
+The path for this resource is made of org-name.vdc-name.edge-name
+For example, using this structure, representing an edge gateway settings that was **not** created using Terraform:
```

Comment:
Strangely, the new line doesn't render between the two lines above.

<img width="870" alt="Screenshot 2020-09-28 at 12 24 08" src="https://user-images.githubusercontent.com/11330689/94414888-89da5f00-0185-11eb-965c-47031719c51e.png">

Reply:
fixed